recipes/pykalman/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.9.5" %}
+
+package:
+  name: pykalman
+  version: {{ version }}
+
+source:
+  fn: pykalman-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pykalman/pykalman-{{ version }}.tar.gz
+  md5: 8baa2921cdfd2a5193ac67e23c2f7e5e
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pykalman
+    - pykalman.datasets
+    - pykalman.sqrt
+    - pykalman.sqrt.tests
+    - pykalman.tests
+
+about:
+  home: http://pykalman.github.io
+  license: BSD License
```

Comment:
I believe it is `BSD 3-Clause`. (See https://github.com/pykalman/pykalman/blob/master/COPYING.)